Sasquatch/Sasquatch.xcodeproj/project.pbxproj

```diff
@@ -2545,7 +2555,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = "";
-				INFOPLIST_FILE = "$(SRCROOT)/SasquatchObjC/Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/SasquatchPuppet/Info.plist";
```

Comment:
It seems to replace content instead of adding a new entry, how does that work?

Reply:
Puppet was Using `SasquatchObjC`'s Info.plist before, so now we have a new entry.

Reply:
Puppet was using `SasquatchObjC/Info.plist` and now it is using the new one.